src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -62,3 +62,135 @@ def body(self):
 
         # test list sql server should be 0
         self.cmd('sql server list -g {}'.format(rg), checks=[JMESPathCheck('length(@)', 0)])
+
+class SqlServerFirewallMgmtScenarioTest(ResourceGroupVCRTestBase):
+
+    def __init__(self, test_method):
+        super(SqlServerFirewallMgmtScenarioTest, self).__init__(__file__, test_method,
+                                                                resource_group='cli-test-sql-mgmt')
+        self.sql_server_name = 'cliautomation03'
+        self.location = "westus"
+        self.administrator_login = 'admin123'
+        self.administrator_login_password = 'SecretPassword123'
+
+    def test_sql_firewall_mgmt(self):
+        self.execute()
+
+    def body(self):
+        rg = self.resource_group
+        loc = self.location
+        user = self.administrator_login
+        password = self.administrator_login_password
+        firewall_rule_1 = 'rule1'
+        start_ip_address_1 = '0.0.0.0'
+        end_ip_address_1 = '255.255.255.255'
+        firewall_rule_2 = 'rule2'
+        start_ip_address_2 = '123.123.123.123'
+        end_ip_address_2 = '123.123.123.124'
+
+        # test create sql server with minimal required parameters
+        self.cmd('sql server create -g {} --server-name {} -l {} '
+                 '--administrator-login {} --administrator-login-password {}'
+                 .format(rg, self.sql_server_name, loc, user, password), checks=[
+                     JMESPathCheck('name', self.sql_server_name),
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('administratorLogin', user)])
+
+        # test sql server firewall create
```

Comment:
I agree with you but personally I like comments. Here tests are truly self explanatory so either we can remove all the comments if we want. I am find with any approach. :)